src/main/java/nextstep/courses/domain/session/FreeSession.java

```diff
@@ -5,16 +5,27 @@
 
 public class FreeSession extends Session {
 
+    private final long fee = 0;
+    private final int maxEnrollments = 0;
+
     public FreeSession(Long id, SessionBody sessionBody, SessionEnrollment sessionEnrollment) {
         super(id, sessionBody, sessionEnrollment);
     }
 
     @Override
     public void enroll(NsUser nsUser, Payment payment) {
         validateSessionStatus();
-        validateDuplicateEnrollment(nsUser);
 
         sessionEnrollment.enrollUser(nsUser);
     }
 
+    @Override
+    public long getFee() {
+        return fee;
```

Comment:
변수 없이 리턴 값을 0으로 고정하면 어떨까요?
```suggestion
    @Override
    public long getFee() {
        return 0;
```

Reply:
네 감사합니다.